manifests/05_deployment.yaml

```diff
@@ -203,7 +203,7 @@ spec:
               echo "Copying system trust bundle"
               cp -f /var/run/configmaps/trusted-ca-bundle/tls-ca-bundle.pem /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
           fi
-          exec /root/manager --log-level=debug
+          exec /usr/bin/cloud-credential-operator operator --log-level=debug
```

Comment:
down below, the container name of 'manager' might make better sense as 'cloud-credential-operator' (or just operator)

Reply:
sure, i'll do this in a follow on PR